### PR TITLE
tests: move docker test to new nightly suite

### PR DESCRIPTION
**Pull request by zyga**
The docker test is currently failing and needs attention (looks like
networking timeout). To get it out of the loop and allow people to write
a more comprehensive and real-world test we need a separate suite that
can run expensive tests overnight.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

**Comment by zyga** [comment]
Aha, I need to patch this to actually not run the nightly suite via travis.

**Comment by fgimenez** [comment]
@zyga yes, you can use `manual: true` at the suite level to prevent the execution unless explicitly requested, see https://github.com/snapcore/spread#manual

Once this lands I can prepare a spread-cron branch for the daily execution.

**Comment by zyga** [comment]
@fgimenez done, thanks.

Although thinking about it perhaps we ought to just exclude the nightly suite in travis. I worry that we will have a manual test in the nightly suite but because of a typo somewhere it will never run (not even overnight).

**Comment by fgimenez** [comment]
@zyga thanks, if the whole suite is going to be excluded from the normal executions I think we should put the `manual` entry at the suite level, as in https://github.com/snapcore/snapd/blob/master/spread.yaml#L328. Other than that LGTM.

File: tests/nightly/docker/task.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 summary: Check that the docker snap works
-
+manual: true
```
Comment (fgimenez) [inline tests/nightly/docker/task.yaml]: yes, it doesn't hurt but no need to keep it here

**Pull request merged**